src/ipolates.F90

```diff
@@ -590,5 +591,87 @@ SUBROUTINE IPOLATES_grib2(IP,IPOPT,IGDTNUMI,IGDTMPLI,IGDTLENI, &
 
   END SUBROUTINE IPOLATES_GRIB2
 
+  !> Special case of ipolates_grib2 when interpolating a single field.
+  !! Removes the km dimension of input arrays so scalars can be passed to ibi/ibo.
+  !!
+  !! @param[in] igdtleni Number of elements of the grid definition
```

Comment:
THe order and names of the @param items have to match the variables in the subroutine argument list.